v1/protocols/ipfs.ts

```diff
@@ -146,7 +146,7 @@ export class IPFSProtocol implements Protocol<Static<typeof IPFSProtocolFields>>
     const pubKey = `ipns://${publishKey}/`
     return {
       enabled: true,
-      link: pubKey,
+      link: `_dnslink.${id}`,
```

Comment:
```suggestion
      link: `ipns://${id}/`,
```

Should just be the domain with `ipns` protocol